src/corelib/corelib.nuspec

```diff
@@ -32,11 +32,11 @@
     <!-- Runtime libraries -->
     <file src="bin\v3.5\$Configuration$\openstacknet.dll" target="lib\net35"/>
     <file src="bin\v3.5\$Configuration$\openstacknet.pdb" target="lib\net35"/>
-    <file src="..\..\docs\Api\v3.5\openstacknet.xml" target="lib\net35"/>
+    <file src="bin\v3.5\$Configuration$\openstacknet.xml" target="lib\net35"/>
```

Comment:
This change will cause the NuGet package to not include XML documentation for methods in the SDK which use `<inheritdoc/>`.